lib/_007/Parser/Syntax.pm

```diff
@@ -170,6 +170,35 @@ grammar _007::Parser::Syntax {
 
     token str { '"' ([<-["]> | '\\\\' | '\\"']*) '"' } # " you are welcome vim
 
+    rule regex-part {
+        <regex-group>
+        [ '|' <regex-group> ]*
```

Comment:
I think it'd be more idiomatic to write this as

    <regex-group>+ % '|'

Unless you know of a good reason not to that I might be missing.

Reply:
Is it something to do with whitespace? Then maybe you need a space before the `+`...

Reply:
Mh, good point. I tend to forgot about this intricacy quite often...

Reply:
Yeah, me too. The details are fuzzy to me, but ISTR it wasn't always like that.